jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt

```diff
@@ -373,7 +373,21 @@ class Colibri2ConferenceHandler(
         if (c2relay.create) {
             val transBuilder = Transport.getBuilder()
             transBuilder.setIceUdpExtension(relay.describeTransport())
-            respBuilder.setTransport(transBuilder.build())
+            c2relay.transport?.sctp?.let {
```

Comment:
Should we throw if SCTP is disabled in the config?

Reply:
It already does, above.